ccp/apis/v1/ccp_server/backend/builds/build_logs.py

```diff
@@ -58,15 +60,11 @@ def response(namespace, appid, jobid, desired_tag, build):
             namespace,
             jenkins_job_name
         ],
-        build_number=build
+        build_number=build_number
```

Comment:
If we want to use `build_number` from the function parameter, we can just use it. Why are we assigning it again?

Reply:
we are not assigning value to the variable, we are passing the variable to the function using the parameter name.

Reply:
But doing this looks like `a=a` to me. Why do we need to do it? We can use `build_number` in the function without doing that as well, no? Would it yield a different result?